actors.md

```diff
@@ -617,26 +594,13 @@ func ProvingPeriodDuration(sectorSize uint64) Integer {
 	// The research team should give us concrete numbers
 	return 24 * 60 * 60 * 2 // number of blocks in one day
 }
-```
-
-### IncreasePledge
 
-Parameters:
-
-- addspace BytesAmount
-
-Return: None
-
-
-```go
-func IncreasePledge(addspace BytesAmount) {
-	// Note: msg.Value is implicitly transferred to the miner actor
-	if miner.Collateral+msg.Value < CollateralForPledge(addspace+miner.PledgeBytes) {
-		Fatal("not enough total collateral for the requested pledge")
-	}
+func ComputeLateFee(power Integer, blocksLate Integer) TokenAmount {
+  return 4 // TODO: real collateral calculation, obviously
```

Comment:
you missed the trailing 2! :)

Reply:
It's implied :P

Reply:
was gonna make a snarky factorial joke but got rekt because 2